src/components/Fullscrn_qrcode.vue

```diff
@@ -3,14 +3,17 @@
     class="mt-3"
     :class="[{ 'bubble-dark': isDarkMode }, { 'bubble-light': !isDarkMode }]"
   >
-    <h3 :class="{ 'text-white': isDarkMode }">Get our mobile app!</h3>
-
-    <div>
-      <p :class="{ 'text-white': isDarkMode }">
-        IOS APP <span :class="{ 'text-white': isDarkMode }"> Andriod APP </span>
-      </p>
+    <h3 :class="{ 'text-white': isDarkMode }">Get The Mobile App!</h3>

```

Comment:
The "T" in "The" should be lowercase (*i.e.*, "t").

Reply:
fixed with 06650cb563424e310cfa3d362f0139ca70253197.